### ISSUE TITLE
Analyse von Kernbereichen des Wing Chun Forums

### ISSUE THREAD

**Issue by Antimon2000** [wingchunforum/Organisation#1]
Analysen zur Bewertung verschiedener Prozesse und Bereiche des Wing Chun Forums sind wichtig, um auf ihrer Grundlage Wachstumshindernisse zu erkennen und notwendige Veränderungen herbeizuführen. Bereiche, in denen diese Analysen durchgeführt werden könnten, sind

- Finanzielle Wachstumsgrenzen innerhalb der derzeitigen Strukturen feststellen
- Mitgliederzufriedenheit ermitteln
- Marketingperformance beurteilen
- Nachwuchsförderung
- etc.

**Comment by boeckers** [comment]
Zu den finanziellen Wachstumsgrenzen:

Hier mal eine vorläufige Analyse, die zusammen mit dem Steuerberater entstanden ist und mir sehr realistisch scheint. Ohne jetzt alle Zahlen und den Berechnungsweg hier ausbreiten zu wollen. Berechnungsgrundlage ist ein Beitrag von 35€ monatlich. (Berücksichtigt werden müssen Kosten, Versicherungen, voraussichtliche Steuern, durchschnittliche Besuchszahlen, durchschnittlich in Anspruch genommene Anzahl der Einheiten, Zahlungsrückstände, und vieles mehr.) 

Die Sicherung des <b> Fortbestandes </b>des WCF ist gegeben, auch dann wenn die Anzahl 
Erwachsenen um 13% oder
Kinder um 100 % auf Null zurückgehen 
Das ist sehr knapp, anders ausgedrückt: weniger als zur Zeit dürfen es nicht werden.

Ein <b> zufriedenstellendes Gehalt <\b> habe ich bei folgendem Anstieg:
Erwachsene um 23,0% und Kinder um 50%  
Das ist bei derzeitiger Auslastung und bei derzeitigen Angebot absolut realistisch. Die Kindergruppen haben noch erheblich größeres Potential. Die Erwachsenengruppen funktionieren dann nur, wenn die Montagsgruppe ebenfalls mindestens 2/3 so gut angenommen wird, wie die anderen Gruppen.

Ein <b> gutes und erstrebenswertes Gehalt, auch langfristig <b> würde ich erhalten bei einem Anstieg 
der Erwachsenen  um 40%
oder
der Erwachsenen um 29% und der Kinder um 150%
Das ist bei derzeitiger Nutzung der Räume nur möglich bei Änderung des Angebotes.

<b> Fazit <\b>
Der nächste Schritt sollte der zum "zufriedenstellenden Gehalt" sein. Dies ist absolut im Rahmen und realistisch. Wenn dann jedoch ein weiterer Ausbau erfolgen soll muss das Angebot erweitert werden bzw. die Anzahl der zu besuchenden Stunden eingeschränkt werden oder ein anderes Preismodell in Kraft treten.

Eine Preiserhöhung auf mind. 35€ in 2016 ist in jedem Falle unausweichlich. 

Nicht mit eingerechnet sind Einnahmen durch Privatunterricht, Kurse und externe Kurse. Der oben genannte Verdienst funktioniert nur, wenn diese Einnahmen bleiben bzw. bestenfalls auch noch ausgebaut werden.

**Comment by boeckers** [comment]
Jetzt stimmen die Hervorhebungen.

**Comment by boeckers** [comment]
Das schwierige ist natürlich, dass die Zahlen sich gegenseitig sehr beeinflussen. Kommen mehr Kinder sinken die Zahlen für die Erwachsenen etc. Kommen zudem Menschen, die sich nicht an den Durchschnitt halten, z.B. 10 Leute treten ein, die jeden Tag kommen, dann entstünden weitere Probleme.

**Comment by boeckers** [comment]
Mit weiteren Standorten entstünde ebenfalls eine vollkommen neue Rechnung.

**Comment by Antimon2000** [comment]
Mit finanziellen Wachstumsgrenzen hatte ich eigentlich ganz andere Sachen gemeint, aber die Zahlen oben sind auch sehr interessant. Sie liefern eine Grundlage, welche wirtschaftlichen Ziele wir uns setzen sollten. Bis wann wollen wir welches Ziel erreichen? Was schaffen wir bis zum Ende des 2. Quartals?

**Comment by boeckers** [comment]
Ich hatte bewusst keinen zeitlichen Rahmen genannt, das stresst mich immer sehr. Ich habe ja zum Glück keinen Sachzwang, so dass uns keine Sachlage zwingt bestimmte Zahlen in bestimmten Zeitabschnitten zu erreichen. Die Zeit ist zum Glück vorbei. Ich wünsche mir aber Ende 2016 mit dem Gehalt wenigstens beim ersten Schritt angekommen zu sein. 

Ich bin jetzt beim nochmaligen durchrechnen jedoch erfreut darüber, dass auch meine finanziellen Ziele sehr realistisch sind. Denn diese Berechnung berücksichtigt ja quasi nur den Gruppenunterricht hier vor Ort. Es sind ja weitere Projekte denkbar.

**Comment by Antimon2000** [comment]
Ursprünglich meinte ich übrigens, dass wir mal überlegen, wo die derzeitige Struktur eigentlich ihre Grenzen hat.

Beispiel Wachstumsfaktor Raum:
Uns steht genau ein Raum zur Verfügung. In diesem können von Montag bis Freitag zwischen 8 und 22 Uhr Veranstaltungen stattfinden. Samstags ggf. halb so lang. Das entspricht einer Bruttodauer von 77 Stunden. Der Raum ist im besten Fall permanent ausgebucht. Falls die Gruppen z.B. maximal 20 Personen umfassen und jede Person im Schnitt 3 Zeitstunden pro Woche da ist, haben wir Kapazitäten für höchstens 513 Schüler. Das ist selbstverständlich nur ein Beispiel. Die wahren Werte (z.B. Teilnehmerzahlen) müssen statistisch ermittelt. Damit liegt auch die obere Schranke deutlich niedriger.

Beispiel Personal:
Du führst den Unterricht bis auf vernachlässigbare Ausnahmen allein durch. Wenn wir also eine 40-Stunden-Woche ansetzen und davon ausgehen, dass du täglich 1 Stunde mit Verwaltungsaufgaben beschäftigt bist, stehen den Schülern pro Woche 35 Unterrichtsstunden zur Verfügung. Damit beschränkt der Faktor Personal die Zahl der betreubaren Schüler (Annahme: 20 Pers. pro Einheit, Teilnahme 3h pro Woche) auf 233.

Vergleicht man nun beide Zahlen, sieht man, dass der Faktor Personal die schärfere obere Schranke impliziert. Daraus ließe sich ableiten, dass wir dort größeren Handlungsbedarf haben.

Insgesamt sind das natürlich nur Beispiele, die besser recherchierter/geschätzter Zahlen bedürfen. Aber auf diese Weise lassen sich Aussagen über den Zustand des WCFs machen. Die Analysen lassen sich dann auch auf weitere Einflussfaktoren (Preise, Preismodell, Werbekosten, ...) ausweiten.

So, ich hoffe, ich habe halbwegs verständlich ausgedrückt, worauf meine meine ursprüngliche Fragestellung abzielte.

**Comment by boeckers** [comment]
Ja, ich verstehe. Sehr gut. Da müssten wir in der Tat eine wesentlich umfangreichere Analyse machen. Ich bin jetzt erstmal von der bisherigen Praxis ausgegangen und habe berechnet, wie wir das Schiff auf die sichere Seite bringen. Nun könnte man aus solchen Zahlen ablesen, das bisher keine Relevanten Hindernisse auftauchen. 

In der Tat wäre aber für das potentielle Wachstum die von die Angedachte Analyse viel mehr auf mögliche Potentiale ausgerichtet.

**Comment by Antimon2000** [comment]
Eine äquivalente Aussage zum ersten Beispiel ist übrigens:

Für jede weitere angebotene Unterrichtsstunde (Zeitstunde) erhöhen wir unsere Kapazität an betreubaren Schülern um 6,66.

**Comment by Antimon2000** [comment]
Ablesen kann man aus den Beispielen noch nicht so viel; die sind ja eher aus der Luft gegriffen. Aber wenn wir verlässlichere Analysen anstellen wollen, könnten wir für den Anfang einfach mal zählen, wie viele Leute in welchen Einheiten anwesend sind. Nach 12 Wochen hätte man sicher schon ein paar ganz gute Zahlen.

**Comment by boeckers** [comment]
Ja, deshalb ja Konjunktiv. Ich meinte, "man könnte aus derlei Zahlen, wenn sie denn solide wären, ablesen dass..." Auch können wir sehen, wo als erstes Probleme auftauchen könnten. Das ist eine gute Idee und Hilft mit auch, bei der Berechnung des notwendigen Beitrags etc. 

Ich beginne zunächst mal eine Statistik mit den Teilnehmerzahlen.

**Comment by Antimon2000** [comment]
Und was genauso wichtig ist: Eine Häufigkeitsverteilung, an der man ablesen kann, wie viele Schüler im Schnitt 1/2/3/... Stunden pro Woche im Unterricht sind!

**Comment by boeckers** [comment]
Ja, das ist sehr wichtig für die Kalkulation und leider auch am schwierigsten nachzuvollziehen. Ich denke mal darüber nach, wie ich vernünftige Daten erheben kann, ohne eine Anwesenheitsliste führen zu müssen.

**Comment by Antimon2000** [comment]
Dann bleibt ja eigentlich nur eine Selbstauskunft, bspw. mittels Fragebogen?

**Comment by Antimon2000** [comment]
Wie waren denn eigentlich die Ergebnisse dieser Statistik?

**Comment by boeckers** [comment]
Da die Seite hier ja theoretisch öffentlich ist, folgen hier nur die Eckdaten.

Berücksichtigt habe ich saisonale Schwankungen und Streuung. Auch habe ich eine monatliche Auswertung für jede Stunde des Stundenplanes vorgenommen. Daraus geht hervor, welche Tage wie stark ausgelastet sind, wo mehr, wo weniger Kapazitäten sind. Hier nur das relevante Ergebnis, was unsere Gesamt-Kapazität angeht:

Wir können ohne Veränderung der räumlichen Situation einen Zuwachs von 31,3% (Zielgruppe Erwachsene) hinnehmen, ohne das wir durch saisonale Schwankungen oder Stoßzeiten unsere Kapazitäten systematisch überschreiten. 
Die Kindergruppen haben ein Potential von 59% Zuwachs (Zielgruppe Kinder).
Das Krafttraining verträgt ohne Umorganisation des für 2016 geplanten Ablaufes 36,5% Zuwachs. Dies entspricht etwas mehr als die Kapazität für die Zielgruppe Erwachsene. Dies wird also nicht erreicht, bevor die Kapazitäten des Raumes bei derzeitigem Stundeplan überschritten ist. 

Für eine finanziell entspannte Situation brauchen wir, bei einem zukünftig geplanten Beitrag von 35€, einen Zuwachs von 30% der Zielgruppe Erwachsene und 25% der Zielgruppe Kinder. Dies ist in der derzeitigen räumlichen Situation und bei derzeitiger Stundenaufteilung möglich und realistisch. Vorausgesetzt jedoch, dass die zukünftige Trainingsgruppe sich, was die Trainingshäufigkeit angeht, so zusammen setzt, wie die bisherige.

Bei Überschreitung der Kapazität gibt es in zeitlicher Reihenfolge folgende Möglichkeiten: 1.) Mehr Einheiten anbieten 2.) Trainingshäufigkeit begrenzen 3.) andere Räumlichkeiten suchen.

**Comment by boeckers** [comment]
Eine weitere Kennzahl, die für mich intern sehr wichtig ist: die Umsatzrentabilität. Hier ohne weitere Erklärung und absolute Zahlen im "halböffentlichen" Raum. Unsere Umsatzrentabilität konnten wir um über 10% steigern. Damit liegen wir richtig gut. Vereinfacht heißt das, das Verhältnis Gewinn zu Umsatz ist extrem gut. Diese Kennzahl zeigt die Effizienz unsere Vorgehensweise und spiegelt, warum wir diese guten Preise nahezu halten können.
